client/src/components/Patients/index.js

```diff
@@ -37,27 +69,64 @@ const Patients = () => {
 
   const columns = [
     {
-      title: 'First Name',
-      dataIndex: 'firstname',
+      title: 'Name',
+      children: [
+        {
+          title: 'First Name',
+          dataIndex: 'firstname',
+        },
+        {
+          title: 'Last Name',
+          dataIndex: 'lastname',
+        },
+      ],
+      responsive: ['xs', 'sm'],
     },
     {
-      title: 'Last Name',
-      dataIndex: 'lastname',
-    },
-    {
-      title: 'Email',
-      dataIndex: 'email',
+      title: 'Age',
+      dataIndex: 'birthday',
+
+      render: (record) => (
+        <>
+          {differenceInCalendarYears(
```

Comment:
it's good to make this subtraction here, but I think will be more fine if we make all of these things and operation in one place ( when we get the data) and then using table only to render values.

we did some data operation [here](https://github.com/GSG-G9/dentoro/blob/fca79d780b7a00f3743a3e0dace047375a3078c2/client/src/components/PatientSearchDateTable/index.js#L50) ,please view it.